packages/core/admin/admin/src/pages/App/index.js

```diff
@@ -84,7 +85,9 @@ function App() {
         if (uuid) {
           const {
             data: { data: properties },
-          } = await axios.get(`${strapi.backendURL}/admin/telemetry-properties`);
+          } = await get(`/admin/telemetry-properties`, {
+            validateStatus: (status) => status < 500,
```

Comment:
I had to pass the validateStatus property because of the interceptors of the fetchClient, as they specify that if a 401 response is given, they redirect to /login. It would end up in an infinite loop when the user doesn't have a session and trying to access the /telemetry-properties route.

Reply:
Could you add this as an online code comment? That's quite a valuable piece of information for any developer who might read this 😊

Reply:
Second this.